### PR TITLE
Data to home page

### PR DESCRIPTION
**Pull request by portmil**

### CODE REVIEW

File: client/src/components/HomePage.js [hunk 1]
```diff
@@ -52,10 +54,13 @@ const HomePage = () => {
...
     }
 
+    // if recipe belongs to multiple categories, use the icon of the first one other than 'All'
+    const icon = recipe.categories.length > 1 ? recipe.categories[1].icon : 'all_icon';
+
     return (
       <div key={recipe.id} className='recipe-card' onClick={() => navigate(`/${recipe.id}`)}>
         <div className='recipe-icon-container'>
-          <img className='recipe-icon' src={allIcon} alt='Icon for all'/>
+          <img className='recipe-icon' src={require(`../graphics/${icon}.svg`)} alt='Icon for the category of the recipe'/>
```
Comment (viivi-uhari) [inline client/src/components/HomePage.js]: Could this alt text have the same format as above? `Icon for category '${category.name}'`

File: client/src/components/HomePage.js [hunk 2]
```diff
@@ -15,11 +16,12 @@ const HomePage = () => {
...
-    setCategories([{icon: 'all_icon', name: 'All'}, {icon: 'breakfast_icon', name: 'Breakfast'}]);
-
-    // get all recipes from backend and replace mock data
-    setRecipes([{id: '1', name: 'Spaghetti bolognese', rating: 3}, {id: '2', name: 'Lentil Soup', rating: 0}]);
+    recipeService.getAll().then(recipes =>
+      setRecipes(recipes)
+    );
+    categoryService.getAll().then(categories =>
+      setCategories(categories)
+    );
```
Comment (viivi-uhari) [inline client/src/components/HomePage.js]: Could a `try catch` structure be implemented here?

File: client/src/components/HomePage.js (hunk 1 above)
Comment (portmil) [reply client/src/components/HomePage.js]: Modified

File: client/src/components/HomePage.js (hunk 2 above)
Comment (portmil) [reply client/src/components/HomePage.js]: Modified

**Pull request merged**